src/components/MaterialUI/textProps.js

```diff
@@ -0,0 +1,22 @@
+import ComponentProps from "./componentProps";
+
+export default class TextProps extends ComponentProps {
+	static propNames = {
+		classes: "classes",
+	};
+
+	static ruleNames = {
+		root: "root",
+		body1: "body1",
```

Comment:
Can we have a better name that "body1" please ?

Reply:
@joclefrancois 
We shouldn't. It corresponds to material ui API (yeah they could create a better name for it but, it's body1)

Reply:
Oh well, so be it